application/view/common/resource-values.phtml

```diff
@@ -32,12 +37,14 @@ $labelInfo = $this->setting('property_label_information');
                 $class[] = 'private';
             }
             ?>
-            <div class="<?php echo implode(' ', $class); ?>" lang="<?php echo $escape($value->lang()); ?>">
-                <?php if ($language = $value->lang()): ?>
-                <span class="language"><?php echo $language; ?></span>
+            <?php if (($filterLocale == 0) || ($valueLang == '') || ($valueLang == $lang)): ?>
+            <div class="<?php echo implode(' ', $class); ?>" lang="<?php echo $escape($valueLang); ?>">
+                <?php if (($showLocale == 1) && ($valueLang)): ?>
+                <span class="language"><?php echo $valueLang; ?></span>
                 <?php endif; ?>
-                <?php echo $value->asHtml(); ?>
+                <?php echo $value->asHtml($lang); ?>
```

Comment:
This is one of the only places (maybe _the_ only place?) where the `$lang` setting gets used regardless of if "filterLocale" is enabled. With it _disabled_ it seems inconsistent that we'd modify the displayed titles of linked resources when listing values (the practical effect here), but not when browsing, etc.

Reply:
Took me some time to figure this out, nice catch. 69e554e